plugins/lookup/onepassword.py

```diff
@@ -42,6 +42,10 @@
         description: The username used to sign in.
       secret_key:
         description: The secret key used when performing an initial sign in.
+      service_account_token:
+        description: The access key for a service account.
```

Comment:
It should be documented that this needs v2 of the CLI tool:
```suggestion
        description:
          - The access key for a service account.
          - Only works with version 2 of the CLI tool.
```